src/plugins/imagery/components/ImageryView.vue

```diff
@@ -58,6 +58,7 @@
             height: `${sizedImageHeight}px`
           }"
           @mousedown="handlePanZoomClick"
+          @dblclick="expand"
```

Comment:
Nice! 

Noticed something: on [line 49](https://github.com/nasa/openmct/pull/7308/files#diff-019c5423367eee44cfe7b8031ca5b554aa21604010b3947ec73eaf4b6058a93eR49), we have a single-click handler registered for clicking on the background. I think we can remove this to keep the action consistent (double-click on the image itself).

Reply:
Thanks. I removed the click event from the `imageBG`. Is that OK?

Reply:
perfect!

Reply:
spoke with @charlesh88 and this change looks good